modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/di/RepositoryProviderModule.kt

```diff
@@ -58,4 +59,8 @@ class RepositoryProviderModule {
     @Singleton
     @ProcessLifecycle
     fun processLifecycle(): LifecycleOwner = ProcessLifecycleOwner.get()
+
+    @Provides
+    @Singleton
+    fun providePreProcessOpmlFile(): PreProcessOpmlFile = PreProcessOpmlFile()
```

Comment:
Why `PreProcessOpmlFile` is injected and as a singleton?

Reply:
I was thinking of preparing `OpmlImportTask` to be more testable, but we don't have any tests for it yet. So I will just create an local instance of `PreProcessOpmlFile`